www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx

```diff
@@ -117,15 +117,23 @@ const getDetailCardLines = ({
     {
       field: details.flapping ? true : undefined,
       line: (
-        <Tooltip title={t(labelResourceFlapping)}>
-          <FlappingIcon color="primary" />
-        </Tooltip>
+        <>
```

Comment:
I would pack everything in one tile instead of having two conditionally. Besides that, I think we'd save up some space by displaying the icon in the same line instead of two.

Reply:
Ok i works on it